src/main/scala/gregtech/common/worldgen/WorldGenRubberTree.java

```diff
@@ -41,6 +41,12 @@ public void generate(Random random, int chunkX, int chunkZ, World world, IChunkG
             BlockSaplingGT sapling = MetaBlocks.SAPLING;
             if(solidBlockState.getBlock().canSustainPlant(solidBlockState, world, randomPos, EnumFacing.UP, sapling)) {
                 BlockPos abovePos = randomPos.up();
+                IBlockState aboveState = world.getBlockState(abovePos);
+                if(!(aboveState.getBlock().isAir(aboveState, world, abovePos) ||
```

Comment:
Dumb check. Top block will always be top solid block, and state above can be only not solid liquid.